src/filelock.c

```diff
@@ -54,11 +54,12 @@ along with GNU Emacs.  If not, see <http://www.gnu.org/licenses/>.  */
 #include "w32.h"	/* for dostounix_filename */
 #endif
 
-
 #ifdef HAVE_UTMP_H
 #include <utmp.h>
 #endif
 
+int mkrstemp(char *template, int flags);
```

Comment:
Maybe you can change the function name to `rust_mkostemp` (to indicate that is from the rust codebase) or something more meaningful.